Makefile

```diff
@@ -15,14 +15,17 @@ run:
 freeze:
 	pip freeze > requirements.txt
 
-test:
-	py.test --cov-report term --cov=$(PROJ_SLUG) tests/
+unit_test:
+	py.test --cov-report term --cov=$(PROJ_SLUG) unit_tests/
+
+integration_test:
+	py.test --cov-report term --cov=$(PROJ_SLUG) integration_tests/
 
 quicktest:
-	py.test --cov-report term --cov=$(PROJ_SLUG) tests/
+	py.test --cov-report term --cov=$(PROJ_SLUG) unit_tests/
 
 coverage:
-	py.test --cov-report html --cov=$(PROJ_SLUG) tests/
+	py.test --cov-report term --cov=$(PROJ_SLUG) integration_tests/ unit_tests/
```

Comment:
actually this will make package stage fail because package runs docs which in turn runs coverage.  lets just drop integration_tests from coverage for now.